src/tests/clipperUI/components/previewViewer/augmentationPreview_tests.tsx

```diff
@@ -34,10 +30,27 @@ export class AugmentationPreviewTests extends TestModule {
 			let defaultComponent = <AugmentationPreview clipperState={mockClipperState} />;
 			MithrilUtils.mountToFixture(defaultComponent);
 
-			Assert.tabOrderIsIncremental([Constants.Ids.highlightButton, Constants.Ids.sansSerif, Constants.Ids.serif, Constants.Ids.decrementFontSize,
+			MithrilUtils.simulateAction(() => {
+				document.getElementById(Constants.Ids.serif).click();
+			});
+			Assert.tabOrderIsIncremental([Constants.Ids.highlightButton, Constants.Ids.serif, Constants.Ids.decrementFontSize,
```

Comment:
As I mentioned, we would want to add tests to ensure that the tabindex does not get set on elements that shouldn't have them. I think these tests will pass if the tabindex gets set on both sans-serif and serif buttons, when they should fail.